microservices/workflow/internal/workflow/shipment.go

```
@@ -0,0 +1,113 @@
+package workflow
+
+import (
+	"encoding/json"
```

golangci-lint (1.64.8)

import 'encoding/json' is not allowed from list 'main': use bytedance/sonic instead of encoding/json

(depguard)